.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:prettier/recommended",
+        "airbnb"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
```

Comment:
I am getting errors like this

![image](https://user-images.githubusercontent.com/28519395/177522403-67e58c63-6bee-4182-9529-302c58647d45.png)

look into the rules and formatting to fix those automatically. But since we are using typescript, we will need to configure specifically for it